SigNoz/signoz-otel-collector
# Update span metrics processor

Fixes https://github.com/SigNoz/signoz-otel-collector/issues/21

How to verify wonder issue is fixed:

Run the v0.12.0 version and set the small [`dimensions_cache size`](https://github.com/SigNoz/signoz/blob/develop/deploy/docker/clickhouse-setup/otel-collector-config.yaml#L77), such as 1, to trigger the issue and notice the services tab and traces with the count. You will see that traces are dropped without exporting when the error happens. 

Now run the same setup with this fix -- the traces should be exported.
